src/main.js

```diff
@@ -278,16 +279,15 @@ export default class Auth {
 		const sessionId = await this.storage.get(this.sKey('SessionId'));
 		// Get the indication if this was a "link account" request.
 		const linkAccount = await this.storage.get(this.sKey('LinkAccount'));
-
-		// Check for the edge case in which the user signed-out
+		// Check for the edge case in which the user signed out
 		// before completing the linkAccount request.
-		if (linkAccount && !this.user) throw Error('Request to "Link account" was made, but user is no longer signed-in');
+		if (linkAccount && !this.user) throw Error('Request to "Link account" was made, but user is no longer signed in');
```

Comment:
Changing error messages should be done separately or with a notice as some users might depend on the exact string.
Since this is a beta I’ll approve it but it’s can usually be considered a breaking change.

Reply:
This is important to know, I didn't know.  Thanks.

Reply:
Though both versions are technically correct, the original is more widely used and therefore should be retained.